### PR TITLE
Added some ubuntu18 tipps for installation.

### PR DESCRIPTION
**Pull request by markuszeller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,12 +87,14 @@ or:
...
 
+See below for more ubuntu18 tipps
+
 ### Homebrew
 
 If you're using [homebrew](https://brew.sh/), you can use the `brew install` command:
 
     brew install exa
-
+
```
Comment (rusty-snake) [inline README.md]: Can be cleared (remove whitespaces).

File: README.md [hunk 2]
```diff
@@ -129,8 +131,25 @@ The first command downloads the virtual machine image, and then runs our provisi
...
 
 Of course, the drawback of having a standard development environment is that you stop noticing bugs that occur outside of it. For this reason, Vagrant isn’t a *necessary* development step — it’s there if you’d like to use it, but exa still gets used and tested on other platforms. It can still be built and compiled on any target triple that it supports, VM or no VM, with `cargo build` and `cargo test`.
+
+### Ubuntu dependencies
+For installing on ubunutu18 try this:
+
+    apt install cmake
+    apt install libssl1.0-dev
+    apt install pkg-config
+    apt install cargo
```
Comment (rusty-snake) [inline README.md]: Maybe easyer to use one apt install (`apt install cmake libssl1.0-dev pkg-config cargo`).

File: README.md [hunk 3]
```diff
@@ -129,8 +131,25 @@ The first command downloads the virtual machine image, and then runs our provisi
...
     All the tests passed!
-
-
+   
 ### Running without Vagrant
 
 Of course, the drawback of having a standard development environment is that you stop noticing bugs that occur outside of it. For this reason, Vagrant isn’t a *necessary* development step — it’s there if you’d like to use it, but exa still gets used and tested on other platforms. It can still be built and compiled on any target triple that it supports, VM or no VM, with `cargo build` and `cargo test`.
+
+### Ubuntu dependencies
+For installing on ubunutu18 try this:
```
Comment (rusty-snake) [inline README.md]: full ubuntu version? 18.04

File: README.md [hunk 4]
```diff
@@ -129,8 +131,25 @@ The first command downloads the virtual machine image, and then runs our provisi
...
+    apt install libssl1.0-dev
+    apt install pkg-config
+    apt install cargo
+    
+    git clone https://github.com/sfackler/rust-openssl.git
+    cd rust-openssl/
+    cargo build
+    
+    git clone https://github.com/rust-lang/git2-rs
+    cd git2-rs
```
Comment (rusty-snake) [inline README.md]: One `cd` is with trailing `/` one without. One `git clone` URL ends with `.git` one not. This looks inconsistent.

File: README.md (hunk 3 above)
Comment (markuszeller) [reply README.md]: yes

**Comment by ogham** [comment]
Hi, thanks for your contribution. I'm more interested in providing official packages rather than build instructions for particular OSes, so I'm not going to merge this.

**Pull request closed without merging**